### PR TITLE
Release 2.0.0 alpha.1

### PR DESCRIPTION
**Pull request by ewinslow**
Haven't tested anything really, but I figure it's an alpha so not expected to be stable.

### CODE REVIEW

**Comment by ewinslow** [comment]
Problem: All unreleased 1.11.x fixes and 1.12 fixes/features are included in this readme and won't be included in the respective readmes for those versions when generated. Do we care?

File: docs/locale/pot/admin.pot [hunk 1]
```diff
@@ -26,2282 +25,1851 @@ msgstr ""
...
 msgid "Getting Help"
 msgstr ""
 
 #: ../../admin/getting-help.rst:4
-# d8baf99e508d4a639ae9fc0b67978654
 msgid "Having a problem with Elgg? The best way to get help is to ask at the `Community Site`_. This site is community supported by a large group of volunteers. Here are a few tips to help you get the help you need."
 msgstr ""
 
 #: ../../admin/getting-help.rst:13
-# 62afaea1a58346d39978e80bc2670ad3
```
Comment (ewinslow) [inline docs/locale/pot/admin.pot]: Nice of transifex to remove these hashes. Was causing a whole lot of useless diff noise...

**Comment by ewinslow** [comment]
@Elgg/owners @Elgg/contributors Should we release 1.11.4, 1.12.0, and 2.0.0-alpha.1 all at once?

**Comment by hypeJunction** [comment]
SGTM, given 1.11.3 never made into the wild.

**Comment by jeabakker** [comment]
Sgtm

**Comment by iionly** [comment]
What's with the "not liked by default" BC breaking change? Wouldn't it make sense to release 2.0.0-alpha.1 with no open BC breaking stuff - "feature freeze"?

**Comment by mrclay** [comment]
I assume @iionly is talking about #8654. I'd like to squeeze that in under the wire even if it creates some regressions in alpha 1 (likes won't show up for comments/bundled stuff).

**Comment by mrclay** [comment]
:+1: to releasing 1.11.4 and 1.12 first.

**Comment by ewinslow** [comment]
We'll release 1.11.4 and 1.12 first. I won't be able to get to this imminently so anyone else can feel free to tackle that if you like...

**Pull request closed without merging**